aiperf/metrics/types/reasoning_token_count.py

```diff
@@ -66,3 +66,9 @@ class TotalReasoningTokensMetric(DerivedSumMetric[int, ReasoningTokenCountMetric
     tag = "total_reasoning_tokens"
     header = "Total Reasoning Tokens"
     short_header = "Total Reasoning"
+    short_header_hide_unit = True
+    flags = (
+        MetricFlags.PRODUCES_TOKENS_ONLY
+        | MetricFlags.LARGER_IS_BETTER
+        | MetricFlags.NO_CONSOLE
```

Comment:
if a model has reasoning tokens, should we present them to the user at the console?

Reply:
@debermudez  i think it could be useful. so could some of the other ones. i dont know how strict we want to stick to the GAP output